website/views/shared/tasks/task.jade

```diff
@@ -133,10 +139,10 @@ li(bindonce='list', bo-id='"task-"+task.id', ng-repeat='task in obj[list.type+"s
               span=env.t('addChecklist')
         form.checklist-form(ng-if='task.checklist')
           fieldset.option-group(ng-if='!$state.includes("options.social.challenges")')
-            legend.option-title
+            legend.option-title(ng-if='task.checklist[0]')
```

Comment:
Sorry @nthomsn, something I've only just noticed: If I'm reading this line correctly, it causes the checklist explanation to be inserted only if a checklist item already exists. Is that correct? If so, it would be better to not use the `ng-if` so that the explanation can be seen all the time. It could help new users work out whether they want to use a checklist or not.